### PR TITLE
feat: add AWS snippets

### PR DESCRIPTION
**Pull request by szhekpisov**

### CODE REVIEW

File: terraform/infrastructure/samples/aws/data.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#   }
+#   tags = {
+#     Tier = "Intra"
+#   }
+# }
+
+data "aws_subnet" "intra_subnet_1a" {
+  filter {
+    name   = "tag:Name"
+    values = ["<CLUSTER_NAME>-vpc-intra-eu-west-1a"]
```
Comment (sergs-pci) [inline terraform/infrastructure/samples/aws/data.tf:21]: please change eu-west-1 to the parameter <CLOUD_REGION> here and in two other subnets bellow

Comment (szhekpisov) [reply terraform/infrastructure/samples/aws/data.tf]: done

File: terraform/infrastructure/samples/aws/sns.tf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# AWS S3 bucket with versioning
+# For more details please see official provider documentation https://registry.terraform.io/modules/terraform-aws-modules/sns/aws/latest
```
Comment (sergs-pci) [inline terraform/infrastructure/samples/aws/sns.tf:2]: suggestion to change this resource 
https://registry.terraform.io/modules/terraform-aws-modules/sqs/aws/latest#subscribe-queue-to-sns-topic

Comment (szhekpisov) [reply terraform/infrastructure/samples/aws/sns.tf]: Changed!

**Review by sergs-pci — APPROVED** [review]

**Review by sergs-pci — APPROVED** [review]

**Pull request merged**